packages/augur-ui/src/modules/modal/reporting.tsx

```diff
@@ -24,8 +25,7 @@ import {
 } from 'modules/contracts/actions/contractCalls';
 
 import Styles from 'modules/modal/modal.styles.less';
-import { createBigNumber } from 'utils/create-big-number';
-import { convertDisplayValuetoAttoValue, Getters } from '@augurproject/sdk';
+import { Getters } from '@augurproject/sdk';
```

Comment:
Same here.

Reply:
There was a format not used Getters is used for putting in `Getters.Accounts.UserCurrentOutcomeDisputeStake`